bigquery/query_property_places.md

```diff
@@ -18,7 +20,9 @@ WHERE Obs.observation_about = 'geoId/06' AND
       Obs.is_preferred_obs_across_facets
 ```
 
-* Latest value from specific source for a single variable, e.g. latest CA population from ACS 5 Year Survey:
+### Latest value from specific source for a single variable
+
+As an example, the latest CA population from ACS 5 Year Survey:
```

Comment:
should we link to https://www.census.gov/programs-surveys/acs ?

Reply:
Done